insights/parsers/ls.py

```diff
@@ -31,6 +31,9 @@
 LSlaZ - command ``ls -lanZ <dirs>``
 -----------------------------------
 
+LSldZ - command ``ls -ldZ <items>``
```

Comment:
It seems the `LSldH` is missed in the previous PR, would you mind adding it here in this PR?